code/game/machinery/Vox.dm

```diff
@@ -0,0 +1,285 @@
+/obj/structure/armalispod
+	name = "large biopod"
+	desc = "A bizarre tangle of meat, chitin and wiring."
+	icon = 'icons/obj/vox.dmi'
+	icon_state = "pod_open_big"
+	anchored = 1
+	density = 1
+	var/mob/living/carbon/human/occupant = null
+	var/used = 0
+
+/obj/structure/armalispod/proc/go_out()
+	if(!occupant)
+		return
+	if (occupant.client)
+		occupant.client.eye = occupant.client.mob
+		occupant.client.perspective = MOB_PERSPECTIVE
+	occupant.forceMove(get_step(loc, SOUTH))
+	occupant = null
+	SetName(initial(name))
+	return
+
+/obj/structure/armalispod/proc/put_mob(mob/living/carbon/M as mob)
+	if(!istype(M))
+		to_chat(usr, "<span class='danger'>The biopod cannot process such a lifeform!")
+		return
```

Comment:
Close this span tag with `</span>`.

Reply:
This entire machine has been removed after some discussion